test/scripts/fixtures/complex.d.ts

```diff
@@ -0,0 +1,917 @@
+/** This d.ts content was partially taken from common/declaration.dts
```

Comment:
which parts did you take? is there a specific reason for choosing one part instead of another? 
I would prefer that d.ts files that we are using for testing will be as short & clear as possible. WDYT?

Reply:
I sampled a bit of the `declaration.d.ts`, I wanted something that includes more that just a module declaration, like a module that uses assets from interfaces and namespaces also declared in the file.

I agree with the short and clear thing, that's way I sampled just two modules